doc/source/whatsnew/v1.0.0.rst

```diff
@@ -231,6 +231,8 @@ Build Changes
 ^^^^^^^^^^^^^
 - Fixed pyqt development dependency issue because of different pyqt package name in conda and PyPI (:issue:`26838`)
 - Added a `pyproject.toml <https://www.python.org/dev/peps/pep-0517/>`_ file (:issue:`20775`)
+- Pandas no longer includes cythonized C files in the source distribution uploaded to PyPI. As long as the version of ``pip``
+  supports reading a ``pyproject.toml``, no changes should be required in the build environment. The does not affect users installing wheels (:issue:`28341`).
```

Comment:
It seems a bit strange to me to put the pyproject / cythonized files note in the "bug fixes" section, I would move it above somewhere in enhancements or changes

Reply:
Moved to a top-level section after enhancements. I removed the pyqt note, since it's a dev-only thing.